### PR TITLE
S-Corp Fishing Rod

### PR DESCRIPTION
**Pull request by ShinyPulsar**
## About The Pull Request
Adds a new fishing rod : Wellcheers fishing rod from S-Corp, obtainable in the rareloot of S-Corp crates. The rod is a combinaison of Gold and Titanium rod with buffed stats (but not too much).

## Why It's Good For The Game
Adds a new rod for fishing, so players don't have to struggle choosing between Gold or Titanium and encourages more to use S-Corp gacha because of it. And of course, makes fishing even better.

## Changelog
:cl:
add: Wellcheers fishing rod
:cl:

### CODE REVIEW

File: ModularTegustation/tegu_items/refinery/crates/corporation.dm [hunk 1]
```diff
@@ -163,6 +163,7 @@
 		/obj/item/gun/ego_gun/shrimp/minigun,
 		/mob/living/simple_animal/hostile/shrimp,
 		/obj/item/grenade/spawnergrenade/shrimp,
+		/obj/item/fishing_rod/wellcheers
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/refinery/crates/corporation.dm:166]:
```suggestion
		/obj/item/fishing_rod/wellcheers,
```

**Comment by Kitsunemitsu** [comment]
I'm merging another S-Corp PR, give me a few mins

**Comment by Kitsunemitsu** [comment]
I moved your rod to the very rare; and am merigng

**Review by Kitsunemitsu — APPROVED** [review]
Approved. Due to how rare it is; this works extrmely well

**Pull request merged**